cloud/terraform.md

```diff
@@ -4,6 +4,7 @@ title: "Terraform"
 keywords: terraform, automation, terraporx, cloud
 sidebar: home_sidebar
 redirect_from: "/terraporx.html"
+meta-description: "Portworx has open-sourced the Terraporx repository, containing Terraform scripts for a number of cloud providers. Check it out!"
```

Comment:
you can ignore that. It was to keep track of where we were in the spreadsheet. This is fine